package.json

```diff
@@ -16,6 +16,7 @@
     "@react-navigation/native": "^6.1.8",
     "@react-navigation/native-stack": "^6.9.14",
     "@supabase/supabase-js": "^2.38.0",
+    "escape-string-regexp": "^5.0.0",
```

Comment:
Let's delete these imports (escape-string-regexp, url-parse) if we don't need them after eslint/prettier is updated!